source/_integrations/music_assistant.markdown

```diff
@@ -33,6 +33,183 @@ Under normal circumstances, Home Assistant automatically discovers your running
+  - **Example**: `playlist`
+- **Data attribute**: `artist`
+  - **Optional**: Yes.
+  - **Description**: When specifying a track or album in the Media ID field, you can optionally restrict results by this artist name.
+  - **Example**: `Queen`
+- **Data attribute**: `album`
+  - **Optional**: Yes.
+  - **Description**: When specifying a track in the Media ID field, you can optionally restrict results by this album name.
+  - **Example**: `News of the world`
+- **Data attribute**: `enqueue`
+  - **Optional**: Yes.
+  - **Description**: If the content should be played now or be added to the queue. Options are:
+    - play: Play now
+    - replace: Replace the existing queue and play now
+    - next: Add to the current queue after the currently playing item
+    - replace_next: Replace the current queue after the currently playing item
+    - add: Add to the end of the queue
+  - **Example**: `replace`
+- **Data attribute**: `radio_mode`
+  - **Optional**: Yes.
+  - **Description**: Enable radio mode to auto-generate a playlist based on the selection.
+  - **Example**: `true`
+
+### Action `music_assistant.play_announcement`
+
+Play announcement on a Music Assistant player with more fine-grained control options.
+
+- **Data attribute**: `url`
+  - **Optional**: No.
+  - **Description**: URL to the notification sound.
+  - **Example**: `https://someremotesite.com/doorbell.mp3`
+- **Data attribute**: `use_pre_announce`
+  - **Optional**: Yes.
+  - **Description**: Use pre-announcement sound. Omit to use the player default.
+  - **Example**: `true`
+- **Data attribute**: `announce_volume`
+  - **Optional**: Yes.
+  - **Description**: Use a forced volume level for the announcement. Omit to use the player default.
+  - **Example**: `75`
+
+### Action `music_assistant.search`
+
+Perform a global search on the Music Assistant library and all providers.
+
+- **Data attribute**: `name`
+  - **Optional**: No.
+  - **Description**: The name/title to search for.
+  - **Example**: `We Are The Champions`
+- **Data attribute**: `media_type`
+  - **Optional**: Yes.
+  - **Description**: The type(s) of content to search for. Select from artist, album, track, radio, or playlist. All types if omitted.
+  - **Example**: `playlist`
+- **Data attribute**: `artist`
+  - **Optional**: Yes.
+  - **Description**: When specifying a track or album in the name field, you can optionally restrict results by this artist name.
+  - **Example**: `Queen`
+- **Data attribute**: `album`
+  - **Optional**: Yes.
+  - **Description**: When specifying a track in the name field, you can optionally restrict results by this album name.
+  - **Example**: `News of the world`
+- **Data attribute**: `limit`
+  - **Optional**: Yes.
+  - **Description**: Maximum number of items to return (per media type).
+  - **Example**: `10`
+- **Data attribute**: `library_only`
+  - **Optional**: Yes.
+  - **Description**: Only include results that are in the library.
+  - **Example**: `true`
```

Comment:
```suggestion
```

search action was (last minute) removed from the PR, awaiting further discussion